content/blog/2022/11/2022-11-17-jenkins-election-candidates.adoc

```diff
@@ -0,0 +1,131 @@
+
+Affiliations: Kainos
+
+link:https://github.com/timja[Github]
+
+=== Events Officer - Alyssa Tong
+
+"It has been a privilege for me to be part of the Jenkins community and serve as Events Officer in 2022.
+I would be honored to serve as Events Officer for another term if given the opportunity.
+I am passionate about open source, collaboration, and education.
+My goal is to build welcoming communities, and coordinating events is part of this role.
+More importantly, I strive to bring the community together through events and conferences, so we can share best practices and collaborate for the betterment of the project."
+
+Affiliations: CloudBees
+
+link:https://github.com/alyssat[Github]
+
+=== Security Officer - Wadeck Follonier
+
+I joined the Jenkins adventure in 2017, as long as the Jenkins Security team and was coached by Oleg Nenashev and Daniel Beck about how to best contribute to the community effort, especially related to security tasks.
+I am working full time at CloudBees, managing two teams, including the security team dedicated to Jenkins.
+I got inspired by Daniel's work in terms of tooling, to create an application helping both the security people and the maintainers through multiple bot commands (in SECURITY tickets and in Hosting request).
+During the year as the Security Officer, I spent a good amount of my time onboarding the future experts, finding ways to increase efficiency of the team by left shifting tasks, to reduce the size of our backlog and to put the focus on important tasks.
+For the next year, I would like to be able to continue the effort to make Jenkins an even more secure place.
+One of the topic that I want to push forward is about the Content Security Policy introduction in the Jenkins ecosystem.
+
+Affiliations: CloudBees
+
+link:https://github.com/Wadeck[Github]
+link:https://www.linkedin.com/in/wadeck/[Linkedin]
+
+=== Documentation Officer - Kevin Martens
+
+I am a technical documentation writer, and have been contributing to the Jenkins project consistently over the last seven months. 
+I have helped create the Blue Ocean status statement that is now in the documentation, DevOps World material, and was a mentor for the Jenkins Documentation area of Hacktoberfest 2022.
+I have recently been added to the copy-editor team due to my involvement with the Jenkins Documentation SIG and contributions.
+
+I have learned constantly by contributing link:/blog/authors/kmartens27/[blog posts] to Jenkins and the link:https://cd.foundation/blog/2022/09/07/jenkins-18th-birthday-%f0%9f%8e%82-and-retrospective/[CD Foundation], documentation updates and creation, and screenshot updates.
+I have also contributed recent LTS release changelogs and upgrade guides, and am currently reviewing the weekly release lines.
+I review open pull requests, issues, and feedback, so that we can discuss further with the community and work to improve things as needed.
```

Comment:
```suggestion
I review open pull requests, issues, and feedback, so that we can discuss further with the community and work to improve as needed.
```